.github/workflows/test_pysteps.yml

```diff
@@ -0,0 +1,108 @@
+name: Test Pysteps installation
+
+env:
+  MINIMAL_DEPENDENCIES: cython numpy jsmin jsonschema matplotlib netCDF4 opencv pillow pyproj scipy dask
+  OPTIONAL_DEPENDENCIES: dask pyfftw cartopy h5py PyWavelets
+  TEST_DEPENDENCIES: pytest pytest-cov codecov
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit_tests:
+    name: Unit Tests (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-latest"] #, "macos-latest", "windows-latest" ]
+        python-version: [ "3.6"] #, "3.8"]
```

Comment:
Why Python 3.6 and not 3.8? It looks like were testing against version 3.8, aren't we? But I may be wrong. ;)

Reply:
I just commented 3.8 out for debugging, now it's back with ad1cb2b